src/v2/Apps/Collect/Utils/collectionToArtistSeriesSlugMap.tsx

```diff
@@ -0,0 +1,183 @@
+export const collectionToArtistSeriesSlugMap = {
+  "alex-katz-departure": "alex-katz-departure",
+  "alex-katz-pas-de-deux": "alex-katz-pas-de-deux",
+  "alex-katz-ada": "alex-katz-ada",
+  "alex-katz-black-dress": "alex-katz-black-dress",
+  "alex-katz-chance": "alex-katz-chance",
+  "alex-katz-coca-cola-girls": "alex-katz-coca-cola-girls",
+  "banksy-barcode": "banksy-barcode",
+  "banksy-choose-your-weapon": "banksy-choose-your-weapon",
```

Comment:
Since slugs can change (on gravity's side), should we use IDs here instead?

Reply:
Old slugs (like these), will still continue to work. If we used id's, would that hurt SEO? We'd redirect to a page with a weird Postgres/uuid in the URL.

Reply:
Ahh you're right, you're right.

Reply:
That being said, we should possibly have something in Force that does a 301/302 for any of our new pages that someone arrives to via id (instead of slug)? (not to implement for artist series, just generally).